extras/mmu/mmu_sync_feedback_manager.py

```diff
@@ -212,14 +223,22 @@ def _handle_sync_feedback(self, eventtime, state):
         if abs(state) <= 1:
             old_state = self.state
             self.state = float(state)
+            self._proportional_seen = True
             self.mmu.log_trace(
                 "MmuSyncFeedbackManager(%s): Got sync force feedback update. State: %s (%s)" % (
                     "active" if self.sync_feedback_enabled and self.active else "inactive",
                     self.get_sync_feedback_string(detail=True),
                     float(state)
                 )
             )
-            self.last_recorded_extruder_position = None # Reset extruder watchdog position
+            # IMPORTANT: Do NOT reset the extruder watchdog every proportional tick.
+            # Only reset on *side* transitions (tension<->compression) so ΔE can accumulate.
+            def _side(v): 
+                return 0 if abs(v) < 1e-3 else (1 if v > 0.0 else -1)
```

Comment:
I'm curious where the `1e-3` / `0.001` comes from. If I understand correctly that the filament pressure sensor is reporting the amount of pressure seen in either direction, this threshold allows for negligible pressure to count as neutral. In which case it makes sense. Do you foresee a need to have that neutral threshold configurable?